packages/time/README.md

```diff
@@ -3,7 +3,44 @@
 The plugin for [Size Limit] to track JS download and execution time by [estimo]
 and Puppeter.
 
-See Size Limit docs for more details.
+## Custom options
+
+By default, Size Limit measures the loading time of your files using a slow 3G network (50 kB/s) without latency. You can customize these settings for each check by modifying your Size Limit configuration:
```

Comment:
We need to keep 80 columns limit

Reply:
Okay, I updated my workspace settings to limit it to 80 columns and updated the doc.

Reply:
Hi @ai, can you please have a look again? :)